handlers/config.go

```diff
@@ -85,6 +93,23 @@ func Config(w http.ResponseWriter, r *http.Request) {
 		},
 	}
 
+	// The following code fetches the cluster info. Cluster info is not critical.
+	// It's even possible that it cannot be resolved (because of Istio not being with MC turned on).
+	// Because of these two reasons, let's simply ignore errors in the following code.
```

Comment:
It's OK to ignore but I also don't see a problem with just an info level log indicating that cluster was not configured.

Reply:
I added several log messages. This should give clarity about what's the issue if no indicator is shown.